contracts/RiskManagerV1.sol

```diff
@@ -214,6 +217,20 @@ contract RiskManagerV1 is Auctioneer, Ownable {
         signerBondsSwapStrategy.swapSignerBonds{value: withdrawableAmount}();
     }
 
+    /// @notice Throws if the state of the deposit is not
+    ///         DEPOSIT_LIQUIDATION_IN_PROGRESS_STATE.
```

Comment:
It does not *throw*, it *reverts* :wink: Also, let's explain why this could happen:
```
    /// @notice Reverts if the deposit for which the auction was created is no
    ///         longer in the liquidation state. This could happen if signer
    ///         bonds were purchased from tBTC deposit directly, outside of
    ///         coverage pool auction.
```

Reply:
Fixed in 2c29566b2fba6d7c26e266b118d46d24f18e4d5c.